doc/deployment-schema.json

```diff
@@ -0,0 +1,343 @@
+                                        "^[a-zA-Z0-9_.-]*$": {
+                                            "type": "object",
+                                            "required": [
+                                                "version",
+                                                "type",
+                                                "status",
+                                                "restartPolicy",
+                                                "settings"
+                                            ],
+                                            "properties": {
+                                                "version": {
+                                                    "type": "string",
+                                                    "examples": [
+                                                        "1.0"
+                                                    ]
+                                                },
+                                                "type": {
+                                                    "$ref": "#/definitions/moduleType"
+                                                },
+                                                "status": {
+                                                    "$ref": "#/definitions/status"
+                                                },
+                                                "restartPolicy": {
+                                                    "$ref": "#/definitions/restartPolicy"
+                                                },
+                                                "env": {
+                                                    "$ref": "#/definitions/env"
+                                                },
+                                                "settings": {
+                                                    "type": "object",
+                                                    "required": [
+                                                        "image",
+                                                        "createOptions"
+                                                    ],
+                                                    "properties": {
+                                                        "image": {
+                                                            "type": "string"
+                                                        },
+                                                        "createOptions": {
+                                                            "$ref": "#/definitions/createOptions"
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "additionalProperties": false
```

Comment:
Do you need this in system modules as well?

Reply:
added